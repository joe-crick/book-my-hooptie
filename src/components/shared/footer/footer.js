import React from "react";
import styles from "./footer.less";
import { translate } from "react-i18next";

export const Footer = ({ t: translate }) => (
  <footer className={styles.footer}>
    <h1 className={styles.text}>{translate("footer")}</h1>
  </footer>
);

export default translate()(Footer);
