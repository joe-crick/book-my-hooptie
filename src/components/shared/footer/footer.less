@import "../../../style-resources/mixins.less";
.footer {
    background-color: @slick-purple;
    margin: @bottom-flush;
    padding: @surround-pad;
    text-align: center;
}

.text {
    color: @almost-white;
    font-size: @small-font;
    font-weight: @light;
}