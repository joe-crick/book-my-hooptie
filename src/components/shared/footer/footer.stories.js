import React from "react";
import { storiesOf } from "@storybook/react";
import { i18nMock } from "test/test-utils";

import { Footer } from "./footer";

storiesOf("Footer", module).add("default", () => <Footer t={i18nMock} />);
