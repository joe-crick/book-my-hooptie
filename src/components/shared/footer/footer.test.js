import React from "react";
import {Footer} from "./footer";
import renderer from "react-test-renderer";
import { i18nMock } from "test/test-utils";

describe("Header", () => {
  it("should render", () => {
    const component = renderer.create(<Footer t={i18nMock} />);
    let header = component.toJSON();
    expect(header).toMatchSnapshot();
  });
});
