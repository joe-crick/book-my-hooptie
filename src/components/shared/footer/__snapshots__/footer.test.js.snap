// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
<footer
  className={undefined}
>
  <h1
    className={undefined}
  >
    footer
  </h1>
</footer>
`;
